source/sass/buyers-guide/views/product.scss

```diff
@@ -4,6 +4,26 @@
 
 .product-detail {
   margin-top: -88px;
+
+  @media (min-width: $bp-md) {
```

Comment:
where is `$bp-md` coming from? (it's not imported anywhere, so if we ever refactor this might suddenly go very wrong)

Reply:
these breakpoint variables are set in `/source/sass/buyers-guide/bg-main.scss`

Reply:
guess we'll have to refactor this a little after launch.

Reply:
@Pomax Do you mind filing a ticket for this? I think we can clean this bit by moving

```scss
$bp-xs: #{map-get($grid-breakpoints, xs)}; // < 576px
$bp-sm: #{map-get($grid-breakpoints, sm)}; // >= 576px
$bp-md: #{map-get($grid-breakpoints, md)}; // >= 768px
$bp-lg: #{map-get($grid-breakpoints, lg)}; // >= 992px
$bp-xl: #{map-get($grid-breakpoints, xl)}; // >= 1200px
```

to its own `.scss` file and `@import` this file into `bg-main.scss` but I'm not sure if this is what you meant. If we do decide to tackle this we'll need to do it for the main foundation site `main.scss` as well. I think filing a ticket to address both cases would be helpful.